### DISCUSSION TITLE
Stop plot already running

### DISCUSSION THREAD

**Discussion by josinaldobarbosa** [swar/Swar-Chia-Plot-Manager#662] [Q&A] [answered]
how to stop a plot that is already running?

**Comment by moon8761** [comment, marked as answer]
win+r cmd
taskkill /f /pid xxxx (you job's pid)

**Reply by k0b1** [reply]
open task manager , go to details find your pid number , end task

**Reply by k0b1** [reply]
don't
 forget delete files from your temporary location...

**Reply by josinaldobarbosa** [reply]
thanks, i already managed to do this, i just asked because i thought i had something to do this directly
